src/components/table/table-column-resize/resizable-table-column.component.html

```diff
@@ -1,4 +1,6 @@
-<ng-content></ng-content>
+<div class="ux-resizable-table-column-content">
```

Comment:
With the new layout there seems to be a few issues if we try to use column sorting as well:
https://plnkr.co/edit/0Z3D2nJT1Ytm6grJ

E.g.:
1. Vertical alignment of column sorter
2. Column sorter gets hidden if column is too small
3. Column sorter is not right aligned

Reply:
The wrapper should go around the text only, something like this:
![image](https://user-images.githubusercontent.com/26114101/93190770-a9ac6480-f73b-11ea-8b49-2745300aabdf.png)

This would be part of the application's template, but we should provide an appropriate class.